README.md

```diff
@@ -135,6 +135,27 @@ See the full [tutorial](docs/tutorial.md) for more details.
 - Open API specification: The Unity Catalog Rest API is documented [here](api).
 - Compatibility and stability: The APIs are currently evolving and should not be assumed to be stable.
 
+## Deployment
+- To create a tarball that can be used to deploy the UC server or run the CLI, run the following:
+  ```sh
+  build/sbt createTarball
+  ```
+  This will create a tarball in the `target` directory.
+- Unpacking the tarball will create the following directory structure:
+  ```
+  unitycatalog-<version>
```

Comment:
this level of detail is not necessary in the readme. Lets make a Deployment section in the main docs. 
- readme will contains just the above step - how to generate a tarball, and then a pointer to the docs page
- docs will contain the full details of the steps to deploy, by untaring, dir structure, how start, which configuration files, etc.

Reply:
I am okay doing that in the different PR though.